libs/seiscomp/seismology/ttt/homogeneous.cpp

```diff
@@ -0,0 +1,129 @@
+/***************************************************************************
+ * Copyright (C) gempa GmbH                                                *
+ * All rights reserved.                                                    *
+ * Contact: gempa GmbH (seiscomp-dev@gempa.de)                             *
+ *                                                                         *
+ * GNU Affero General Public License Usage                                 *
+ * This file may be used under the terms of the GNU Affero                 *
+ * Public License version 3.0 as published by the Free Software Foundation *
+ * and appearing in the file LICENSE included in the packaging of this     *
+ * file. Please review the following information to ensure the GNU Affero  *
+ * Public License version 3.0 requirements will be met:                    *
+ * https://www.gnu.org/licenses/agpl-3.0.html.                             *
+ *                                                                         *
+ * Other Usage                                                             *
+ * Alternatively, this file may be used in accordance with the terms and   *
+ * conditions contained in a signed written agreement between you and      *
+ * gempa GmbH.                                                             *
+ ***************************************************************************/
+
+#include <seiscomp/core/strings.h>
+#include <seiscomp/math/geo.h>
+#include <seiscomp/seismology/ttt/homogeneous.h>
+
+#include <cmath>
+#include <vector>
+
+using namespace std;
+
+namespace
+{
+
+double radToDeg(double r) { return 180.0 * r / M_PI; }
+double degToRad(double d) { return M_PI * d / 180.0; }
+
+double computeDistance(double lat1,
+                       double lon1,
+                       double lat2,
+                       double lon2,
+                       double *azimuth = nullptr,
+                       double *backAzimuth = nullptr)
+{
+	double dist, az, baz;
+	Seiscomp::Math::Geo::delazi(lat1, lon1, lat2, lon2, &dist, &az, &baz);
+	dist = Seiscomp::Math::Geo::deg2km(dist);
+	if (azimuth) *azimuth = az;
```

Comment:
We use consequently spaces within parenthesis: `if ( azimuth )`

Reply:
In general we are about to change the code towards curly brackets even for blocks with only one statement to avoid indentation errors. We ran several times into issues with that.

```cpp
if ( backAzimuth ) {
    *backAzimuth = baz;
}
```